src/Umbraco.Web.UI.Client/src/views/components/buttons/umb-toggle-group.html

```diff
@@ -3,10 +3,11 @@
         <umb-toggle class="umb-toggle-group-item__toggle"
                     checked="item.checked"
                     disabled="item.disabled"
+                    input-id="{{$id}}-{{$index}}"
```

Comment:
Maybe something like this would be better?

```
input-id="{{item.inputId}}"
```

and then use `String.CreateGuid()` in controller.

It would be a bit more consistent with this:
https://github.com/umbraco/Umbraco-CMS/commit/124415e413a7f716b220a126513d1b967da73cff

and e.g. `<umb-checkbox>`
https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Web.UI.Client/src/common/directives/components/forms/umbcheckbox.directive.js#L52

The `<umb-toggle>` set an id here:
https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Web.UI.Client/src/common/directives/components/buttons/umbtoggle.directive.js#L78

But in these changes it overwrite the id.

It could probably extend each item in the directive to set something like this:
```
item.inputId = "umb-toggle-group-item_ + String.CreateGuid();
```

https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Web.UI.Client/src/common/directives/components/buttons/umbtogglegroup.directive.js#L73-L86

Reply:
Thank you, @bjarnef. I planned to do this, but I spent an inordinate amount of time because I am not familiar with AngularJS conventions, and I did not know that `inputId` is referenced as `input-id` in the view. Anyway, I made an update based on your feedback. If anything else could be improved, please let me know.